.github/workflows/deploy.yaml

```diff
@@ -40,24 +64,31 @@ jobs:
         run: |
```

Comment:
Hva er | for her hvis kommandoen tar bare en linje?

Reply:
Kunne vært en linje ja 👍